app/dependencies.py

```diff
@@ -1,10 +1,42 @@
-# Dependency
+from fastapi import Depends, HTTPException, status
+from fastapi.security import OAuth2PasswordBearer
+from jose import jwt
+from pydantic import ValidationError
+from sqlalchemy.orm import Session
+
+from app.schemas import schemas_tokens
+from app.core import security
+from app.core.settings import settings
+from app.cruds import cruds_users
+from app.models import models_users
+
 from app.database import SessionLocal
 
+reusable_oauth2 = OAuth2PasswordBearer(tokenUrl="/auth/token")
```

Comment:
Why do you call the schema reusable ?

Reply:
It's the name used by FastAPI.
https://fastapi.tiangolo.com/tutorial/security/oauth2-jwt/